env-template

```diff
@@ -14,3 +14,14 @@ DEBUG=True
 # ALLOWED_HOSTS =  json.loads(config('ALLOWED_HOSTS'))
 ALLOWED_HOSTS='["127.0.0.1", "192.168.99.100", "localhost"]'
 
+# Your Login and Password to MAIL  settings.py  should be as follow:
+# EMAIL_HOST_USER = config("EMAIL_HOST_USER", default="default@email.com")
+# EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="password")
+# EMAIL_HOST = config("EMAIL_HOST", default="smtp.gmail.com")
+# EMAIL_HOST_PORT = config("EMAIL_HOST_PORT", default=587)
+# EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=True)
```

Comment:
I don't think that this comment is needed. Everything is already in settings and self-explanatory. So please remove.